pkg/controller/nodes/task/k8s/plugin_manager.go

```diff
@@ -399,14 +418,18 @@ func NewPluginManager(ctx context.Context, iCtx pluginsCore.SetupContext, entry
 		Type: entry.ResourceToWatch,
 	}
 
-	ownerKind := iCtx.OwnerKind()
 	workflowParentPredicate := func(o metav1.Object) bool {
+		if entry.OverrideInjectOwnerReferences != nil && *entry.OverrideInjectOwnerReferences {
```

Comment:
can we not add a label? I guess but as step 1 this is fine. This may be too chatty?

Reply:
Talked IRL. When I think the idea would be to target this in a follow-up PR? right @kumare3?